data/json/items/ammo.json

```diff
@@ -19,6 +19,18 @@
     "//2": "1 battery is 1 kJ of energy. 40 batteries is an alkaline C-cell.",
     "fuel": { "energy": 1 }
   },
+  {
+    "type": "AMMO",
+    "id": "rock",
```

Comment:
Why are you adding second "rock" item? There is already one existing, and I doubt the staff could be throwing 6L-volume rocks.

Reply:
going to be changing that

Reply:
This has been the holdup for slings for a very long time, making seperate versions of mundane items to transform them into sling ammunition is not ok.

Reply:
There is now only one rock item, no seperate version